handlers/tx.go

```diff
@@ -13,9 +15,32 @@ import (
 	"github.com/tendermint/tendermint/types"
 	"golang.org/x/net/context"
 	"gopkg.in/mgo.v2/txn"
+	"strings"
 	"time"
 )
 
+var _parser msgparser.MsgParser
+
+func InitRouter(conf *config.Config) {
+	router := msgparser.RegisteRouter()
```

Comment:
RegisteRouter when it is judged that OnlySupportModule is empty.
```suggestion
	var router msgparser.Router
```